data_model/src/isi.rs

```diff
@@ -1190,12 +1206,8 @@ pub mod error {
             #[display(fmt = "Invalid WASM binary: {_0}")]
             Wasm(String),
             /// Invalid name length (i.e. too long [`AccountId`])
-            #[display(
-                fmt = "Name must be between {} and {} (inclusive) characters in length",
-                "_0.start()",
-                "_0.end()"
-            )]
-            NameLength(RangeInclusive<u32>),
+            #[display(fmt = "Name length violation")]
```

Comment:
Why have you decided to simplify this error message? Wouldn't it will be useful for user to know exact limits on error.

Reply:
Because `Range` doesn't implement `PartialEq`

Reply:
We could implement `PartialOrd` for the entire struct manually, and still have the extra information. I still think that this should be a separate type.

Reply:
opened an [issue](https://github.com/hyperledger/iroha/issues/3590)